service/src/services/dispatchers/in-app/in.app.dispatcher.ts

```
@@ -0,0 +1,107 @@
+      const receiversCount = data.reduce(
+        (acc, value) => acc + value.receiverIDs.length,
+        0
+      );
+      this.logger.verbose(
+        `Dispatching ${data.length} in-app compressed notification payloads for a total of ${receiversCount} receivers`,
+        LogContext.IN_APP_DISPATCHER
+      );
+    }
+
+    try {
+      return this.sendWithoutResponse(data);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)